lib/validation.tsx

```diff
@@ -106,26 +108,61 @@ const isFieldResponseValid = (
       break;
     }
     case "fileInput": {
+      //TODO need refactoring.
       const typedValue = value as { file: File; src: FileReader; name: string; size: number };
-      if (validator.required && typedValue.file === null) return t("input-validation.required");
+      if ((validator.required && typedValue === undefined) || typedValue === null)
+        return t("input-validation.required");
+      if (validator.required && typedValue && typedValue.file === null)
+        return t("input-validation.required");
       // Size limit is 8 MB
-      if (typedValue.size > 8000000) return t("input-validation.file-size-too-large");
+      if (typedValue && typedValue.size > 8000000) return t("input-validation.file-size-too-large");
```

Comment:
Please limit the scope of your PR to the issue you are solving. If you are touching on other code in your PR give the team a heads up so as to avoid merge conflicts

Reply:
This was an issue that i found went uploading a file in a dynamic Row. And i added a fix. i also had a chat with Clement so that he was aware of it. I will try not  next time.

Reply:
All good thanks for adding context. Could you post in forms-dev next time so we're all aware.